src/main/java/org/sonarlint/intellij/util/HelpLabelUtils.kt

```diff
@@ -28,10 +28,24 @@ class HelpLabelUtils {
     companion object {
 
         @JvmStatic
-        fun createCleanAsYouCode() =
+        fun createCurrentFileHelpNotConnected() =
                 ContextHelpLabel.createWithLink(null,
-                        "Deliver clean code by focusing on code that was recently modified",
-                        "Learn more about Clean as You Code", true) { BrowserUtil.browse(FOCUS_ON_NEW_CODE_LINK) }
+                        "SonarLint analyzes in real-time the active file only and reports any issues found in this view. " +
+                            "SonarCloud complements SonarLint by detecting more across the whole project.",
+                        "Try SonarCloud for free", true) {
+                    LinkTelemetry.SONARCLOUD_SIGNUP_PAGE.browseWithTelemetry()
+                }
+
+        @JvmStatic
+        fun createCurrentFileHelp() =
+            ContextHelpLabel.create(
+                "SonarLint analyzes in real-time the active file only and reports any issues found in this view.")
```

Comment:
You can probably extract the text in a variable instead of duplicating it twice. I expect we will always use the same text